### ISSUE TITLE
3.5.2-alpine OpenSSL broken

### ISSUE THREAD

**Issue by carbolymer** [docker-library/python#128]
Invocation of pip3 install is impossible on 3.5.2-alpine image:
```
$ docker build .
Sending build context to Docker daemon 271.9 kB
Step 1 : FROM python:3.5.2-alpine
 ---> 9a646c8ffe06
Step 2 : WORKDIR /si
 ---> Using cache
 ---> f9776040eaf0
Step 3 : COPY requirements.txt /tmp
 ---> Using cache
 ---> 6a27c18de034
Step 4 : RUN python3 -c 'import ssl; print(ssl.OPENSSL_VERSION)'
 ---> Running in 5ad2d70780c4
OpenSSL 1.0.2h  3 May 2016
 ---> d54a691e78cc
Removing intermediate container 5ad2d70780c4
Step 5 : RUN pip3 install --index-url http://pypi.python.org/simple --trusted-host pypi.python.org --requirement /tmp/requirements.txt
 ---> Running in 70dcb10acb68
Collecting scipy==0.17.1 (from -r /tmp/requirements.txt (line 1))
  Downloading scipy-0.17.1.tar.gz (12.4MB)
    Complete output from command python setup.py egg_info:
    Download error on https://pypi.python.org/simple/numpy/: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:645) -- Some packages may not be found!
    Couldn't find index page for 'numpy' (maybe misspelled?)
    Download error on https://pypi.python.org/simple/: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:645) -- Some packages may not be found!
    No local packages or download links found for numpy>=1.6.2
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-rhh3r_cn/scipy/setup.py", line 265, in <module>
        setup_package()
      File "/tmp/pip-build-rhh3r_cn/scipy/setup.py", line 262, in setup_package
        setup(**metadata)
      File "/usr/local/lib/python3.5/distutils/core.py", line 108, in setup
        _setup_distribution = dist = klass(attrs)
      File "/usr/local/lib/python3.5/site-packages/setuptools/dist.py", line 269, in __init__
        self.fetch_build_eggs(attrs['setup_requires'])
      File "/usr/local/lib/python3.5/site-packages/setuptools/dist.py", line 313, in fetch_build_eggs
        replace_conflicting=True,
      File "/usr/local/lib/python3.5/site-packages/pkg_resources/__init__.py", line 826, in resolve
        dist = best[req.key] = env.best_match(req, ws, installer)
      File "/usr/local/lib/python3.5/site-packages/pkg_resources/__init__.py", line 1092, in best_match
        return self.obtain(req, installer)
      File "/usr/local/lib/python3.5/site-packages/pkg_resources/__init__.py", line 1104, in obtain
        return installer(requirement)
      File "/usr/local/lib/python3.5/site-packages/setuptools/dist.py", line 380, in fetch_build_egg
        return cmd.easy_install(req)
      File "/usr/local/lib/python3.5/site-packages/setuptools/command/easy_install.py", line 634, in easy_install
        raise DistutilsError(msg)
    distutils.errors.DistutilsError: Could not find suitable distribution for Requirement.parse('numpy>=1.6.2')

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-rhh3r_cn/scipy/
The command '/bin/sh -c pip3 install --index-url http://pypi.python.org/simple --trusted-host pypi.python.org --requirement /tmp/requirements.txt' returned a non-zero code: 1

```
pip should use HTTP, not HTTPS in such case. I've tried the same command on the 3.5.2-sim image - it works

**Comment by caduvieira** [comment]
ca-certificates not installed by default according to #109

**Comment by tianon** [comment]
Fixed by #136
